packages/@okta/vuepress-theme-prose/assets/css/components/_code.scss

```diff
@@ -27,3 +27,20 @@ pre[class*="language-"] .code-copy {
   white-space: normal;
   color: #333333;
 }
+
+.highlight-lines {
+  user-select: none;
+
+  position: absolute;
+
+  top: 0;
+  left: 0;
+
+  padding: 0.5rem 0 0.5rem 0;
+
+  width: 100%;
+
+  .highlighted {
+    background-color: rgba(0 0 0 / 0.66);
```

Comment:
Just checked how we show code blocks in dark theme, we practically don't highlight them...

Two asks:
- Can you also take a look at dark theme colour?
- Can we use a variable here and in code blocks? It will be exactly the same value for all cases and it will be easy to update.

A more user friendly option:
<img width="916" alt="image" src="https://github.com/okta/okta-developer-docs/assets/4158731/0e60f0d0-2d2d-4409-8173-9d42dd2c3df6">
The current style:
<img width="962" alt="image" src="https://github.com/okta/okta-developer-docs/assets/4158731/f4bdbd69-bd64-47aa-baf5-94f162f6d37b">

Reply:
Plus one to this. Even the dark mode example on the Vuepress site is _incredibly_ hard to read.

Reply:
I have changed the dark theme colors a bit. 
Attached is the image showing how code blocks with highlighted lines will look in the new theme (both light and dark theme are shown)

**Light**:
<img width="991" alt="Screenshot 2024-07-03 at 12 25 28 PM" src="https://github.com/okta/okta-developer-docs/assets/154415906/5554fcab-b55e-4cc2-b047-46d01498c208">


**Dark**:

<img width="1026" alt="Screenshot 2024-07-03 at 12 25 33 PM" src="https://github.com/okta/okta-developer-docs/assets/154415906/78928039-1d44-4814-87d6-249688ccbd12">